components/org.wso2.carbon.identity.recovery.endpoint/src/main/resources/api.identity.recovery.yaml

```diff
@@ -582,7 +582,7 @@ definitions:
           $ref: '#/definitions/Property'
 
 #-----------------------------------------------------
-# The Error Response  object
+# The Error Response object
```

Comment:
Shall we use a consistent method.. like each word's first letter upper case

Reply:
I see this as "Error Response" is the name of the object and hence it is title case in a sentence that would otherwise be sentence case.